design/reboot-interface.md

```diff
@@ -89,22 +89,34 @@ section of the BareMetalHost status. This records a time after which the Host
 was last booted using the current image. Processes running prior to this time
 may be assumed to have been stopped.
 
-A new date-time field, ``lastReboot``, will be added to the ``provisioning``
+A new date-time field, ``lastRebootRequest``, will be added to the ``provisioning``
 section of the BareMetalHost status. This records a time after which the Host
-was last requested to reboot.
+was last requested to reboot (because we cannot trust any value from the user,
+who even if well intentioned, may have created the timestamp on a machine that 
+was not synchronised with the cluster or has a different timezone).
 
 Since the user interface requirements are still unclear, we will follow 
 standard practices of using an annotation ( ``host.metal3.io/reboot`` ) to 
-trigger reboots.  
+trigger reboots.  It shall contain the UID of the target Machine so that the
+controller can validate that the associated Machine has not changed since the
+annotation was created.
```

Comment:
That's not the reason; it's because the controller needs to remove any reboot annotation it placed once the Machine no longer has the healthcheck annotation, and it needs to be able to tell annotations it placed apart from those that might have been put there by other actors (maybe completely unrelated to healthcheck).